libraries/package-extractor/src/tests/PackageExtractor.test.ts

```diff
@@ -0,0 +1,155 @@
+import path from 'path';
+
+import { FileSystem, Terminal, StringBufferTerminalProvider } from '@rushstack/node-core-library';
+import { loadRushProjectFromConfigurationFile } from './Utils';
+import {
+  PackageExtractor,
+  type IExtractorProjectConfiguration,
+  type IExtractorDependencyConfiguration
+} from '../PackageExtractor';
+
+describe(PackageExtractor.name, () => {
+  // because this files will be compiled to js, and the path will change if using __dirname
+  // using path inside src folder
+  const testRepoRoot = path.join(path.resolve(__dirname, '../../'), 'src/tests/package-extractor-test-repo');
+  const testRepoBaseTargetFolder = path.join(testRepoRoot, 'output');
+  const projects = loadRushProjectFromConfigurationFile(path.resolve(testRepoRoot, 'rush.json'));
```

Comment:
This should be passed in via an argument instead of parsing it again here.

Reply:
I'm not sure how can it be passed in...As it's inside jest test function.
Could you please provide more details?